addons/logistics_wirecutter/functions/fnc_destroyFence.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: BaerMitUmlaut
+ * Destroys the given fence and replaces it with a destroyed fence if possible.
+ *
+ * Arguments:
+ * 0: Fence <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [fence] call ace_logistics_wirecutter_fnc_destroyFence
+ *
+ * Public: No
+ */
+params ["_fence"];
```

Comment:
Blank line above `params`.

Reply:
Why?

Reply:
For consistency with most of the other function files.

Reply:
Should put it in the guidelines then. There's a bunch of functions which don't have a blank line there. I can change it here, ofc.